compiler/passes/scopeResolve.cpp

```diff
@@ -1918,7 +1959,26 @@ static bool lookupThisScopeAndUses(BaseAST* scope, const char * name,
         forv_Vec(ModuleSymbol, mod, *modules) {
           if (mod) {
             if (Symbol* sym = inSymbolTable(mod->block, name)) {
-              symbols.push_back(sym);
```

Comment:
I was worried about that as well.  What led me to keep it separate was the hope that when the ability to be private is more prevalent, this shared code will shrink to nearly nothing and so the sharing would be unnecessary.  But then I'll have two places to simplify instead of just one . . . but it shouldn't be a maintenance issue because it should be fairly short-lived . . .